tests/DMSDocumentTest.php

```diff
@@ -205,6 +222,50 @@ public function testCanEdit()
     }
 
     /**
+     * Tests delete permissions
+     */
+    public function testCanDelete()
+    {
+        $this->logoutMember();
+        $document1 = $this->objFromFixture('DMSDocument', 'doc-logged-in-users');
+
+        // Logged out user test
+        $this->assertFalse($document1->canDelete());
+
+        // Test editors can delete
+        $contentAuthor = $this->objFromFixture('Member', 'editor');
+        $this->assertTrue($document1->canDelete($contentAuthor));
+    }
+
+    /**
+     * Tests create permission
+     */
+    public function testCanCreate()
+    {
+        $this->logoutMember();
+        $document1 = $this->objFromFixture('DMSDocument', 'doc-logged-in-users');
+        $this->logInWithPermission('CMS_ACCESS_DMSDocumentAdmin');
+        // Test CMS access can create
+        $this->assertTrue($document1->canCreate());
+
+        $this->logoutMember();
+
+        // Test editors can create
+        $contentAuthor = $this->objFromFixture('Member', 'editor');
+        $this->assertTrue($document1->canCreate($contentAuthor));
+    }
+
+    /**
+     * Logs out any active member
+     */
+    protected function logoutMember()
+    {
+        if ($member = Member::currentUser()) {
+            $member->logOut();
+        }
```

Comment:
Just put this into `setUp` - there's no harm in running it for the tests that don't need it. Then you can remove its references.